### PR TITLE
Implement Harris's awesome design

### LINKED ISSUE #8: I click the wrong answer button a lot
Better labels?  A separate submit step after you click a button?   Larger/differently laid out buttons?

### PR DESCRIPTION
**Pull request by ruthie**
Current visual below!

Things I'm not sure about:
* I have now four different CSS files, one for each component, except I put the instruction text styles in the file with the play button styles because I was tired of making new files.  Not sure this matters much, but I'm not sure how the best way to do this is.
* I feel weird about passing around classnames to get the answer button layout to work correctly.  I was doing it by passing around the rows and columns, but that felt even worse.

It's fun how quickly I can tweak some things to make them look way better!

Screen Shot 2020-03-28 at 10 54 56 AM

Closes #8

### CODE REVIEW

File: src/AnswerButton.css [hunk 1]
```diff
@@ -1,11 +1,87 @@
+.answer-buttons-container  {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows:  7;
+    grid-column-gap: 20px;
+    grid-row-gap: 10px;
+}
+
+.answer-button {
+    border-radius: 1000px; /* any large number of pixels gets the desired effect */
```
Comment (ruthie) [inline src/AnswerButton.css:10]: It feels weird to use pixel numbers here, but if I use percentages, it gives me an oval shaped button,  which is not what I'm aiming for!  Is there a better way to do this?

File: src/IntervalExercise.js [hunk 2]
```diff
@@ -4,19 +4,77 @@ import Tone from "tone";
...
     return (
-        <button onClick={onClick}> Play Sound </button>
+        <button onClick={onClick} className='play-button'>
+            <MdVolumeUp className='play-button-sound-icon'/>
+            PLAY INTERVAL
+        </button>
     )
 }
 
+const intervalDisplayInfo = {
```
Comment (harrislapiroff) [inline src/IntervalExercise.js]: Style note: I'd probably consider this a true constant and capitalize it `INTERVAL_DISPLAY_INFO`

File: src/IntervalExercise.js [hunk 3]
```diff
@@ -4,19 +4,77 @@ import Tone from "tone";
 import Teoria from "teoria";
 
 import './AnswerButton.css';
+import './PlayButton.css';
+import { MdVolumeUp } from 'react-icons/md';
 
-function AnswerButton({ text, onClick, value, color }) {
+function AnswerButton({ text, onClick, value, color, layoutClassName}) {
```
Comment (harrislapiroff) [inline src/IntervalExercise.js]: Style note: For consistency, if something is going to be applied as an html class, I usually just call the prop `className`. (I do this even in cases like this one where you'll also be adding additional classes yourself.)

File: src/IntervalExercise.js [hunk 4]
```diff
@@ -4,19 +4,77 @@ import Tone from "tone";
...
+        </button>
     )
 }
 
 function PlaySoundButton({ onClick }) {
     return (
-        <button onClick={onClick}> Play Sound </button>
+        <button onClick={onClick} className='play-button'>
+            <MdVolumeUp className='play-button-sound-icon'/>
+            PLAY INTERVAL
```
Comment (harrislapiroff) [inline src/IntervalExercise.js]: In general I would capitalize this like standard English in the JSX, e.g., `Play Interval`, and then use the css declaration `text-transform: uppercase` to apply the stylistic capitalization.

File: src/PlayButton.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    margin-right: 30px;
+}
+
+.instructions-text {
+    font-size: 18pt;
+    font-family: inherit;
+    font-weight: bold;
+    color: #095EB8;
+    text-align: left;
+    letter-spacing: .1em;
```
Comment (harrislapiroff) [inline src/PlayButton.css]: Aesthetic opinion: Personally, I don't usually like fiddling with the letter spacing on standard cased letters.

File: src/Quiz.js [hunk 6]
```diff
@@ -2,15 +2,17 @@ import React, { Component } from "react";
 import IntervalExercise  from './IntervalExercise.js'
 import PropTypes from "prop-types";
 
+import './ContinueButton.css';
+import './Quiz.css';
+
 function ContinueButton({ onClick }) {
     return (
-        <button onClick={onClick}>Continue</button>
+        <button onClick={onClick} className='continue-button'>CONTINUE</button>
```
Comment (harrislapiroff) [inline src/Quiz.js]: See above comment about capitalization

File: src/AnswerButton.css (hunk 1 above)
Comment (harrislapiroff) [reply src/AnswerButton.css:10]: Hm, no, I think this is actually the correct way to do this, or hard code the actual exact value.

**Review by harrislapiroff — APPROVED** [review]
Overall this looks great! I'm super impressed with how well you implemented my design and it's clear you've learned a lot about CSS in the process! 😮 

I've got a bunch of notes for you, mostly because you're still in the early part of the learning curve, but overall I think this is a great improvement even if you choose to overlook some of my comments or just merge as it.

Also one thought is that it might make sense to start learning about SASS/SCSS, though I'm not sure if that comes baked into `create-react-app` or not.

**File Structure**
In a typical React app I would divide this up into more files. Typically I have one `js` and one `css` file per react component (i.e., `AnswerButton` and `PlaySoundButton` would have their own CSS/JS files). You can also put this off for later though.

**Responsive Design**
It seems like you're optimizing this for desktop but based off of the mobile design I made. The result is that the design looks like a mobile design, but it doesn't actually lay out properly on a mobile screen. Again, fine for now, but you'll probably want to change that eventually. (Though it occurs to me now that your design might be perfect now for tablets!) You can simulate the size of mobile screens easily using the responsive design mode in most browsers' web inspectors. If you want to actually optimize for both mobile and desktop you'll want to start learning about [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/@media).

image

**Typography**
For better consistency and predictability, I prefer to define my font sizes in `px` instead of `pt`s. Some people feel strongly about using `em`s and `rem`s, but I'm not one of them. `pt`s are a measurement designed for print. It's like measuring things on a website in `in`—CSS technically allows it but almost no one does it. (Unless you're designing a print stylesheet, in which case it's totally appropriate!)


You have only included the light version of Quicksand on the page, so when you try to use `font-weight: bold` you're forcing the browser to artificially inflate the letterforms. Here's what the looks like

Artificial | Medium (500) | Bold (800)
----|-----|-----
Screen Shot 2020-04-08 at 7 58 59 PM | Screen Shot 2020-04-08 at 8 01 19 PM | Screen Shot 2020-04-08 at 8 00 40 PM

The difference is subtle, but it matters for polish. (If you want to see it with your own eyes, look for the way the horizontal bit of the stroke on curves is narrower than the vertical bits on the artificial one or how the ends of the `I` look less round than on the real variants.) You'll want to go back to the [Google Fonts page](https://fonts.google.com/specimen/Quicksand) and select all the variants you plan to use.

Also, since Quicksand comes in a wider spectrum of weights than normal/bold it might be good to be in the habit of using numbers to indicate font weight, e.g., `font-weight: 300`. In CSS 300 is light, 400 is normal, and 700 is bold. There's also a number of possible options between or around those.

**Pull request merged**